openvpn/tunnelUp.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-if [ "${PEER_DNS}" != "no" ]; then
+if [ "${PEER_DNS}" != "false" ]; then
```

Comment:
I would just check if true.. should work to just check the variable directly and interpret it as a bool

Reply:
No argument there, I've just been following the style w/in that file... I've generally not looked through the other code until now.  Related to that, this should be `[[ "${PEER_DNS,,}" == true ]]`; for case reasons.

Pushing an update shortly to address this.